warehouse/models/intermediate/payments/int_payments__refunds.sql

```diff
@@ -0,0 +1,88 @@
+{{ config(materialized = "table") }}
+
+WITH
+
+micropayments_table_refunds AS (
+    SELECT
+
+        aggregation_id,
+        micropayment_id,
+        participant_id,
+        customer_id,
+        ABS(charge_amount) AS refund_amount,
+        EXTRACT(DATE FROM transaction_time AT TIME ZONE "America/Los_Angeles") AS transaction_date,
+        _line_number,
+        SAFE_CAST(currency_code AS NUMERIC) AS currency_code,
```

Comment:
I actually wonder if this `SAFE_CAST` should happen in the staging table? Normally casting should happen in staging